COPYRIGHT.md

```diff
@@ -0,0 +1,3 @@
+# Copyright
+
+- Copyright (c) 2023-2024 Leon Budnick
```

Comment:
I do not think that this list is complete, but it is a good start. 😄

Reply:
if one wants to be included they can add themself via a pr